tests/test_game_runner.py

```diff
@@ -36,48 +35,48 @@ def get_int(self):
 
 @pytest.fixture
 def game_runner():
-    return GameRunner(game=MockGame(), output=MockInterface())
+    return GameRunner(game=MockGame(), interface=MockInterface())
 
 
 def test_returns_a_game():
     game = MockGame()
-    runner = GameRunner(game=game, output=MockInterface())
+    runner = GameRunner(game=game, interface=MockInterface())
     assert runner.get_game() == game
 
 
-def test_returns_an_output():
-    output = MockInterface()
-    runner = GameRunner(game=MockGame(), output=output)
-    assert runner.get_output() == output
-
-
-def test_play_turn_requests_current_board_state(game_runner):
-    game_runner.play_turn()
-    assert game_runner.get_game().get_board_state_call_count == 1
+def test_returns_the_interface():
+    interface = MockInterface()
+    runner = GameRunner(game=MockGame(), interface=interface)
+    assert runner.get_interface() == interface
 
 
-def test_play_turn_requests_board_to_be_printed(game_runner):
-    game_runner.play_turn()
-    assert game_runner.get_output().render_board_call_count == 1
+def test_run_requests_current_board_state_each_turn_and_on_game_over(game_runner):
```

Comment:
I like that you're testing your inner loop. This is pretty tricky to do given that you have various side effects with each iteration of the loop and indirect assertions are a great way to do test this.

How might your tests look if, instead of asserting on the number of times a method is called, you assert on the behaviour that method is used for? I believe here you are checking that the board is printed to the console? What about something like:

```
console = MockConsole()
runner = GameRunner(console)
runner.run()
assert console.times_display_output_called(10)
```